src/server/rest/v1/service/AuthorizationService.ts

```diff
@@ -154,18 +218,18 @@ export default class AuthorizationService {
     const authorizations: AuthorizationFilter = {
       filters: {},
       dataSources: new Map(),
-      projectFields: [
-        'site.id', 'site.name', 'site.address.city', 'site.address.country', 'siteAdmin', 'siteOwner', 'userID'
-      ],
-      authorized: userToken.role === UserRole.ADMIN,
+      projectFields: [],
+      authorized: false,
     };
     // Filter projected fields
     authorizations.projectFields = AuthorizationService.filterProjectFields(authorizations.projectFields, filteredRequest.ProjectFields);
     // Handle Sites
-    await AuthorizationService.checkAssignedSiteAdminsAndOwners(tenant, userToken, null, authorizations);
+    await this.canPerformAuthorizationAction(tenant, userToken, Entity.USER_SITE, Action.LIST,
+      authorizations, filteredRequest, null, true);
     return authorizations;
   }
 
+  // ASSIGN/UNASSIGN USERS TO/FROM SITE
```

Comment:
Capslock ? :D

Reply:
no, i was just shouting at usersitesiteuser stuff 🤣 
✅ modified in next commit